apps/templates/includes/sidebar.html

```diff
@@ -96,15 +96,33 @@
               </li>
             </ul>
           </li>
-          <li class="nav-item">
-            <a href="{{url_for('home_blueprint.running_labs')}}" class="nav-link {% if request.url_rule.endpoint == 'home_blueprint.running_labs' %} active {% endif %}">
-              <i class="nav-icon far fa-calendar-alt"></i>
```

Comment:
it seems like you accidentally remove these lines here. Can you please re-insert them?